language/en/common.php

```diff
@@ -36,6 +37,7 @@
 	'IDEAS'					=> 'Ideas',
 	'IDEA_DELETED'			=> 'Idea successfully deleted.',
 	'IDEA_LIST'				=> 'Idea List',
+	'IDEA_NO'				=> 'Idea #',
```

Comment:
I think I'd rather this be `IDEA_NUM` as the current one makes it seem like it's for the idea being denied.